test/units/TEST-74-AUX-UTILS.busctl.sh

```diff
@@ -53,6 +53,12 @@ busctl emit --auto-start=no --destination=systemd-logind.service \
             /org/freedesktop/login1 org.freedesktop.login1.Manager \
             PrepareForShutdown b false
 
+systemd-run --service-type=notify --unit=test-busctl-wait busctl --timeout=3 wait /test org.freedesktop.fake1 TestSignal
+busctl emit /test org.freedesktop.fake1 TestSignal s success
+
+(! systemctl is-failed test-busctl-wait.service )
+journalctl -u test-busctl-wait --grep 's "success"'
```

Comment:
If the log level of the service manager is debug, then `journalctl -u` may not work as expected, as you can see in the CI failure. See comments in TEST-04-JOURNAL.invocation.sh.
Maybe, calling `systemctl log-level info` before this test works, hopefully.

Reply:
why would debug cause the message to be missed?

Reply:
If PID1 logs something before creating symlink about unit info under /run/systemd/units, journald caches process information without the unit info, and the log message and subsequent are stored to journal by using the cached data, hence journal entries miss unit information.

Reply:
Tried to just skip the journal, see the new test.